javascript/index.js

```diff
@@ -0,0 +1,87 @@
+function Book(bookTitle, bookAuthor) {
+  this.bookTitle = bookTitle;
+  this.bookAuthor = bookAuthor;
+}
+
+function DisplayBook() {
+  this.books = [];
+}
+
+DisplayBook.prototype.addBook = function (book) {
+  this.books.push(book);
+  this.displayBook(book);
+  this.saveBooksToLocalStorage();
+};
+
+DisplayBook.prototype.displayBook = function (book) {
+  const bookList = document.getElementById('bookList');
+  const div = document.createElement('div');
+  div.classList.add('book');
+
+  const title = document.createElement('div');
+  title.textContent = `Title: ${book.bookTitle}`;
+  div.appendChild(title);
+
+  const author = document.createElement('div');
+  author.textContent = `Author: ${book.bookAuthor}`;
+  div.appendChild(author);
+
+  const deleteBtn = document.createElement('button');
+  deleteBtn.textContent = 'Delete';
+  deleteBtn.classList.add('delete-btn');
+  div.appendChild(deleteBtn);
+
+  bookList.appendChild(div);
+};
+
+DisplayBook.prototype.clearForm = function () {
+  document.getElementById('title').value = '';
+  document.getElementById('author').value = '';
+};
+
+DisplayBook.prototype.deleteBook = function (bookTitle, bookAuthor) {
+  // eslint-disable-next-line max-len
+  this.books = this.books.filter((book) => !(book.bookTitle === bookTitle && book.bookAuthor === bookAuthor));
+  this.saveBooksToLocalStorage();
+};
```

Comment:
- [ ] the delete function is also deleting the wrong book. The  oldest book on the UI is the one that gets deleted. Please add an id field to the books and keep in unique then use this Id field to target the book to be deleted 👍🏾

Reply:
Hello @wayungi 
Thank you for taking the time to review our work. However, I wanted to bring to your attention that the specific code you mentioned does not exist anymore. Based on the feedback from the previous review, we made some revisions and updates which might have resulted in the removal or renaming of certain sections of the code. Therefore, it is possible that you may have made a mistake during the review. If you could kindly provide us with further details or clarify your concerns, we would be more than happy to address them accordingly.